### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by mariapetrosyann** [vladsargsyan/MLProject#2]
Hi,

The project is fantastic. The visualization, the model training and in general Neural Network is so good. The accuracy is hight the loss is low and everything is perfect.

Grate job!

Best regards,
Maria
